exe/fil-tools/cmd/deals_retrieve.go

```diff
@@ -0,0 +1,84 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"io"
+	"os"
+	"path"
+
+	"github.com/caarlos0/spin"
+	"github.com/ipfs/go-cid"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	retrieveCmd.Flags().StringP("address", "a", "", "wallet address of the file to fetch")
+	retrieveCmd.Flags().StringP("cid", "c", "", "cid of the file to fetch")
```

Comment:
Nit: maybe te concept of _file_ can be avoided, in general. Not sure though.

Reply:
Sure. Changed to "data".